docs/accessibility.md

```diff
@@ -8,13 +8,13 @@ to ensure that our application is accessible to every possible user.
 
 ###### Lint Checks
 
-These are fairly simple static analysis checks. You're probably familiar with warning pop-ups when you have small code style issues; in Lockbox for Android, the build will fail if you do not address these warnings!
+These are fairly simple static analysis checks. You're probably familiar with warning pop-ups when you have small code style issues; in Lockbox for Android, the build will fail if you do not address these warnings! 
 
 The most important accessibility warning is the `contentDescription` warning on images. This warning will appear when the `contentDescription` attribute-- the text description of the image --has not been set.
 
 ###### Espresso Tests
 
-`AccessibilityChecks.enable()` -- This should be added in setup methods for your Espresso tests.
+```AccessibilityChecks.enable()``` -- This should be added in setup methods for your Espresso tests. 
```

Comment:
I'm curious, why the triple ticks?

Reply:
I have no idea where the triple ticks came from 🤔 I didn't touch the espresso section. Odd.. maybe a formatting error? Taking them out.